pkg/ccl/backupccl/targets.go

```diff
@@ -404,6 +404,12 @@ func fullClusterTargetsBackup(
 	return fullClusterDescs, fullClusterDBIDs, nil
 }
 
+// selectTargets loads all descriptors from the selected backup manifest(s),
```

Comment:
nit:

selectTargets loads all descriptors from the selected backup manifest(s), and filters the descriptors based on the targets specified in the restore. Post filtering the method returns:
- A list of all descriptors (table, type, database, schema) along with their parent databases.
- A list of explicitly requested database descriptors.
- A list of tenants to restore, if applicable.

The current comment has some misleading statements such as `restoring into a multi-tenant cluster` so I think making it a little more general will work better if the code changes in the future?

Reply:
@adityamaru i like my original comment related to the second bullet , "a list of database descriptors IFF the user is restoring on the cluster or database level",   because the databases aren't really "explicitly" requested in a cluster restore. What do you prefer?

Reply:
yeah sounds good to me 👍

Reply:
done